utils/check_modules.sh

```diff
@@ -44,12 +44,11 @@ fi
 
 /bin/rm -f pylint.log
 
-./utils/sl/check_script.py -s
-
-# Test the build code with daos_build.
-PYTHONPATH=$PWD/utils
+PYTHONPATH=$PWD/site_scons/
```

Comment:
I have a small enhancement. We can calculate the absolute path to `check_modules.sh` file. So, we will be able to run this script from any other path on the system. Otherwise, the relative paths might point to an incorrect location, even `$PWD`

`CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`

and then

`PYTHONPATH=${CURRENT_DIR}/site_scons/`
`${CURRENT_DIR}/utils/sl/check_script.py -s`

and so on

Reply:
Done.